pytube/contrib/channel.py

```diff
@@ -148,12 +184,23 @@ def _extract_videos(raw_json: str) -> Tuple[List[str], Optional[str]]:
         # this is the json tree structure, if the json was extracted from
         # html
         try:
-            videos = initial_data["contents"][
-                "twoColumnBrowseResultsRenderer"][
-                "tabs"][1]["tabRenderer"]["content"][
-                "sectionListRenderer"]["contents"][0][
-                "itemSectionRenderer"]["contents"][0][
-                "gridRenderer"]["items"]
+            try:
```

Comment:
I think it could be simplified into something like
```
try:
    # Possible tabs: Home, Videos, Shorts, Live, Playlists, Community, Channels, About
    active_tab = {}
    for tab in initial_data["contents"]["twoColumnBrowseResultsRenderer"]["tabs"]:
        if tab["tabRenderer"].get("content"):
            active_tab = tab
            break
    # This is the json tree structure for videos
    videos = active_tab["tabRenderer"]["content"]["richGridRenderer"]["contents"]

    # This is the json tree structure of visitor data
    # It is necessary to send the visitorData together with the continuation token
    self._visitor_data = initial_data["responseContext"]["webResponseContextExtensionData"][
        "ytConfigData"]["visitorData"]

except (KeyError, IndexError, TypeError):
```